lib/services/alexa/request/lifecycle/initialize.ts

```diff
@@ -34,6 +34,7 @@ export const initializeGenerator = (utils: typeof utilsObj) => async (runtime: A
   const { stack, storage, variables } = runtime;
 
   storage.delete(S.STREAM_TEMP);
+  storage.delete(S.GO_TO_REF);
```

Comment:
there was an edge case if the skill just ended, this would persist to the next session, and if the choice block is the first choice block, all the choices are ignored